doc/release-notes.md

```diff
@@ -66,9 +66,12 @@ Segwit also simplifies the ability to add new features to Bitcoin and
 improves the efficiency of full nodes, which will help provide long-term
 benefits to Bitcoin users.
 
-Activation for the segwit soft fork is being managed using BIP9 versionbits.
-The earliest activation date is set to 15 November 2016 (UTC) and
-its versionbit is bit 1.
+Activation for the segwit soft fork is being managed using BIP9
+versionbits.  Segwit's version bit is bit 1, and nodes will begin
+tracking which blocks signal support for segwit on 15 November 2016
```

Comment:
Still not technically correct. Tracking will start from the first retarget after the 15th.